crates/egui_extras/src/datepicker/button.rs

```diff
@@ -101,6 +105,20 @@ impl<'a> DatePickerButton<'a> {
         self.highlight_weekends = highlight_weekends;
         self
     }
+
+    /// Set the start year for the date picker. (Default: None)
+    #[inline]
+    pub fn start_year(mut self, start_year: i32) -> Self {
+        self.start_year = Some(start_year);
+        self
+    }
+
+    /// Set the end year for the date picker. (Default: None)
+    #[inline]
+    pub fn end_year(mut self, end_year: i32) -> Self {
+        self.end_year = Some(end_year);
+        self
+    }
```

Comment:
I suggest you simplify this with a single call with a `RangeInclusive`. I suspect that if you set one, you want to set both.

You should also explain what it is for (the dropdown of years you can choose from)